validate_sptx/cli.py

```diff
@@ -0,0 +1,28 @@
+import argparse
+import json
+import sys
+
+from starfish.util.argparse import FsExistsType
+from validate_sptx.validate_sptx import validate
+
+
+class Cli:
+    parser_group = None
+
+    @staticmethod
+    def add_to_parser(parser):
+        parser.add_argument(
+            "--experiment-json",
+            required=True,
+            metavar="JSON_FILE_OR_URL")
+        parser.add_argument(
+            "--fuzz", action="store_true")
+        parser.set_defaults(starfish_command=Cli.run)
+        Cli.parser_group = parser
+
+    @staticmethod
+    def run(args, print_help=False):
+        try:
```

Comment:
Why catch KeyboardInterrupt here (I'm Interested, not questioning the decision).

Reply:
If not, then long running commands will throw massive stack traces when the user hits `Ctrl-C`.

Reply:
Got it -- good to know. Thanks!